test/OpenTelemetry.Instrumentation.GrpcClient.Tests/OpenTelemetry.Instrumentation.GrpcClient.Tests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Unit test project for OpenTelemetry GrpcClient instrumentation</Description>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="Proto\greet.proto" GrpcServices="Client;Server" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="Moq" Version="4.11.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+
+    <PackageReference Include="Google.Protobuf" Version="3.12.3" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.25.0" />
```

Comment:
I think you should update to 2.30.0. It resolves race conditions between the response task being completed and the listener being written to.

Otherwise looks good to go.

Reply:
👍 Yep did the trick. I updated to 2.30.0-pre1 since it appears that 2.30.0 has not been published for all the dependencies yet. I can come back and update this again when they have been published.

Reply:
Uh oh. Hehe, seems we're back to the same build error I was experiencing a few days back. You mentioned there may be other ways to resolve this?

Reply:
I think, we can ignore the Azure pipeline. i think we will just use Github Actions for now...
@cijothomas , what do you think?